eng/pipelines/common/templates/runtimes/wasm-runtime-and-send-to-helix.yml

```diff
@@ -83,6 +83,6 @@ steps:
           # DotNet-HelixApi-Access variable group
           helixAccessToken: $(HelixApiAccessToken)
 
-        helixProjectArguments: '$(coreClrRepoRoot)/tests/helixpublishwitharcade.proj'
+        helixProjectArguments: '$(Build.SourcesDirectory)/src/coreclr/tests/helixpublishwitharcade.proj'
```

Comment:
Unrelated to your PR as such but this is actually a change we might want to make globally. There was a time during the consolidation when the variable was useful as it could have two values at some point but that is no more and the fewer variables we have, the easier we can reason about the pipeline logic.